packages/lib-user/src/components/Contributors/helpers/getExportData.js

```diff
@@ -0,0 +1,76 @@
+import { arrayOf, number, shape, string } from 'prop-types'
+
+export function getExportData ({
+  stats,
+  projects,
+  users
+}) {
+  // create a list of all project names
+  let privateProjectIndex = 1
+  const projectNames = [...new Set(
+    stats.group_member_stats_breakdown.flatMap(member => {
+    return member.project_contributions.map(statsProject => {
+      const project = projects.find(project => project.id === statsProject.project_id.toString())
+      return project?.display_name || `Private Project ${privateProjectIndex++}`
```

Comment:
Currently, in zooniverse/front-end-monorepo/blob/lib-user_csv-export/packages/lib-user/src/components/Contributors/components/ContributorsList/ContributorsList.js#L51 it just notes "Private Project" for any and all private project display names, but should it include an incremented index there too? I lean towards yes, but not sure.

Reply:
Addressed in e0c38ab .